### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by cedricdcc**

### CODE REVIEW

**Comment by bulricht** [comment]
tests are not running

File: pyrdfstore/store.py [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+        log.debug(f"from SPARQLStore :: {type(result)=} -> {result=}")
+        return result
+
+    def insert(self, graph: Graph, named_graph: Optional[str] = NIL_NS):
+        assert (
+            self.allows_update
+        ), "data can not be inserted into a store if no write_uri is provided"
+        log.debug(f"insertion of {len(graph)=} into ({named_graph=})")
+        store_graph = Graph(store=self.sparql_store, identifier=named_graph)
+        try:
```
Comment (marc-portier) [inline pyrdfstore/store.py]: are we sure we want this ? 
this hides runtime failures as it demotes them to mere log-file-entries

**Comment by cedricdcc** [comment]
Will make a separate branch where the tests will run and merge it into this one once it does

File: tests/test_store.py [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+@pytest.mark.usefixtures("prepopulated_rdf_store")
+def test_select_property_trajectory(prepopulated_rdf_store: RDFStore):
+    # SPARQL to select trajectory of a property
+    sparql = """
+    SELECT ?s ?o WHERE {?s <http://purl.org/dc/terms/abstract>/<http://purl.org/dc/terms/else> ?o .}
+    """
+
+    prepopulated_rdf_store.select(sparql)
+    # we should just get here without an error
+    assert True
```
Comment (marc-portier) [inline tests/test_store.py:263]: strange, no?
maybe at least assert the query-result is not none?
or (even better) make sure the prepopulated store has triples on board that guarantee some useful response-count to be checked for?

File: tests/test_store.py [hunk 3]
```diff
@@ -0,0 +1,335 @@
...
+        SELECT ?s
+        WHERE {
+          [] dcat:resource ?s .
+        }
+    """
+    prepopulated_rdf_store = rdf_store
+    print(prepopulated_rdf_store)
+    prepopulated_rdf_store.select(sparql)
+    # we should just get here without an error
+    assert True
```
Comment (marc-portier) [inline tests/test_store.py:284]: similarly - make the test more meaningful

File: tests/test_store.py [hunk 4]
```diff
@@ -0,0 +1,335 @@
...
+
+    sparql = """PREFIX dcat: <http://www.w3.org/ns/dcat#> SELECT ?s WHERE {
+                   [] dcat:resource ?s .
+               }
+
+    """
+
+    prepopulated_rdf_store.select(sparql)
+    assert True
```
Comment (marc-portier) [inline tests/test_store.py:295]: similarly - make the test more meaningful

File: tests/test_store.py [hunk 5]
```diff
@@ -0,0 +1,335 @@
...
+
+@pytest.mark.usefixtures("rdf_store")
+def test_skolemize_fail(rdf_store: RDFStore):
+
+    graph = Graph()
+    graph.parse(str(TEST_INPUT_FOLDER / "3293.jsonld"), format="json-ld")
+
+    rdf_store.insert(graph)
+    assert True
+
```
Comment (marc-portier) [inline tests/test_store.py:320]: similarly - make the test more meaningful

File: try-store.py
Comment (marc-portier) [inline try-store.py:1]: do we need this file? 
seems a remant of eronous local file commit
whatever it does (or did) should now be in the tests?

**Review by marc-portier — CHANGES_REQUESTED** [review]
pls perform minor cleanups
for the more elaborate tests -- ok if we mark that in a follow up issue as a TODO

File: try-store.py
Comment (cedricdcc) [reply try-store.py:1]: No general purpose anymore , so deleted

File: tests/conftest.py [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+
+@pytest.fixture()
+def rdf_store():
+    read_uri = os.getenv("TEST_SPARQL_READ_URI", None)
+    write_uri = os.getenv("TEST_SPARQL_WRITE_URI", None)
+    print(f"read_uri: {read_uri}")
+    print(f"write_uri: {write_uri}")
+    return create_rdf_store(read_uri, write_uri)
+    if read_uri is not None:
```
Comment (marc-portier) [inline tests/conftest.py]: lines 20 to 23 are unreachable and make no more sense -- @cedricdcc please remove this historic glitch before merge

File: tests/test_store.py [hunk 7]
```diff
@@ -0,0 +1,340 @@
+#! /usr/bin/env python
+from logging import getLogger
+from pathlib import Path
+from time import sleep
+from typing import List, Tuple
+from uuid import uuid4
+
+import pytest
+from rdflib import BNode, Graph, Literal, Namespace, URIRef
+from util4tests import run_single_test
```
Comment (marc-portier) [inline tests/test_store.py]: should also import log
```python
from util4tests import run_single_test, log
```

File: tests/test_store.py [hunk 8]
```diff
@@ -0,0 +1,340 @@
...
+
+from pyrdfstore.store import RDFStore
+
+log = getLogger("tests")
```
Comment (marc-portier) [inline tests/test_store.py]: should not be redefined here but imported from `util4tests`

**Pull request merged**